timely_beliefs/beliefs/utils.py

```diff
@@ -644,7 +659,17 @@ def read_csv(
     if cumulative_probability is not None:
         df["cumulative_probability"] = cumulative_probability
 
-    return classes.BeliefsDataFrame(df, sensor=sensor)
+    # Construct BeliefsDataFrame
+    bdf = classes.BeliefsDataFrame(df, sensor=sensor)
+
+    # Apply transformations
+    if transformations:
```

Comment:
I think we can remove this `if` by setting the default value of `transformations` to `[]` (an empty array).

Reply:
You've stumbled upon a little Python boobytrap. See https://stackoverflow.com/questions/366422/how-can-i-avoid-issues-caused-by-pythons-early-bound-default-parameters-e-g-m